machine/machine_core/ssh_connection.py

```diff
@@ -260,7 +260,7 @@ def __execution_opts(self, direct=False):
             return ["-o", "ControlPath=" + self.ssh_master]
 
     def execute(self, command, input=None, environment={},
-                stdout=None, quiet=False, direct=False, timeout=120,
+                stdout=subprocess.PIPE, quiet=False, direct=False, timeout=120,
```

Comment:
I checked `git grep -A10 '\.execute' test | grep stdout=` in all our projects to spot potential fallout, but we use that argument very rarely. The only user is in cockpit test/common/testlib.py:

    m.execute("journalctl|gzip", stdout=fp)

which is fine.

However, we use it in two places in bots itself:
```
image-create:                self.machine.execute(f"/var/tmp/SETUP {self.machine.image}",
image-create-                                     environment=env, stdout=sys.stdout, quiet=not self.machine.verbose, timeout=7200)

vm-run:            machine.execute(cmd, stdout=1)
```

Could you please change them to `stdout=None`? That's not a behaviour change, but would look a little cleaner, in the spirit of this commit.